content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,141 @@ Use VoIP services in Odoo with OnSIP
+   administrative dashboard will appear. In the left menu, click on :guilabel:`Users` and then
+   select the user. By default the user will open on the :guilabel:`User Info` tab. Click on the
+   :guilabel:`Phone Settings` tab to reveal  OnSIP configuration credentials. In the first column,
+   last in the list of credentials.
+
+Fill in the following fields with the associated credentials listed below:
+
+- :guilabel:`SIP Login` / :guilabel:`Browser's Extension` = OnSIP :guilabel:`Username`
+- :guilabel:`OnSIP authorization User` = OnSIP :guilabel:`Auth Username`
+- :guilabel:`SIP Password` = OnSIP :guilabel:`SIP Password`
+- :guilabel:`Handset Extension` = OnSIP :guilabel:`Ext.` (extension without the `x`)
+
+.. image:: onsip/onsip-creds.png
    :align: center
+   :alt: Onsip user credentials with username, auth username, SIP password, and extension
+         highlighted.
+
+.. tip::
+   The OnSIP extension can be found in the :guilabel:`User` banner line above the tabs.
+
+Once these steps are complete, and the work is saved, Odoo users can make phone calls by clicking
+:guilabel:`📞 (phone)` icon  in the top right corner of Odoo.
 
-You can now make phone calls by clicking the phone icon in the top right corner of Odoo (make sure
-you are logged in as a user properly configured in Odoo and in OnSIP).
+.. seealso::
+   Additional setup and troubleshooting steps can be found on `OnSIP's knowledgebase
+   <https://support.onsip.com/hc/en-us>`_.
 
-.. note::
-   If you see a *Missing Parameters* message in the Odoo softphone, make sure to refresh your Odoo
-   window and try again.
+Incoming calls
+--------------
+
+The Odoo database will also receive incoming calls that produce pop-up windows in Odoo. Click the
+green :guilabel:`📞 (phone)` icon to answer the call.
+
+   .. image:: onsip/incoming-call.png
+      :align: center
+      :alt: alt text
+
+.. seealso::
+   :doc:`voip_widget`
+
+Troubleshooting
+---------------
+
+.. tip::
+   If a *Missing Parameters* message apprears in the Odoo softphone, make sure to refresh the Odoo
+   browser window (or tab) and try again.
 
    .. image:: onsip/onsip04.png
       :align: center
+      :alt: Missing parameter message in the Odoo VoIP widget.
 
-.. note::
-   If you see an *Incorrect Number* message in the Odoo softphone, make sure to use the
-   international format, leading with the plus (+) sign followed by the international country code.
-   E.g.: `+16506913277` (where `+1` is the international prefix for the United States).
+.. tip::
+   If an *Incorrect Number* message appears in the Odoo softphone, make sure to use the
+   international format for the number. This means leading with the :guilabel:`+ (plus)` sign
+   followed by the international country code.
+
+   A country code is a locator code that allows access to the desired country's phone system. The
+   country code is dialed first, prior to the target number. Each country in the world has its own
+   specific country code.\
```

Comment:
I can't tell if the backslash here is used for formatting purposes, because when I ran make, it didn't seem to show up in the build. If so, feel free to ignore it, but if not, I wanted to call it out in case it was a typo.